specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/databox.json

```diff
@@ -0,0 +1,4685 @@
+          "description": "Device type to be used for the job.",
+          "enum": [
+            "DataBox",
+            "DataBoxDisk",
+            "DataBoxHeavy"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsExtensible": true,
+            "values": [
+              {
+                "value": "DataBox",
+                "description": "Data Box."
+              },
+              {
+                "value": "DataBoxDisk",
+                "description": "Data Box Disk."
+              },
+              {
+                "value": "DataBoxHeavy",
+                "description": "Data Box Heavy."
+              }
+            ]
+          }
+        }
+      },
+      "x-ms-discriminator-value": "ValidateCreateOrderLimit"
+    },
+    "CreateOrderLimitForSubscriptionValidationResponseProperties": {
+      "description": "Properties of create order limit for subscription validation response.",
+      "required": [
+        "validationType"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ValidationInputResponse"
+        }
+      ],
+      "properties": {
+        "status": {
+          "description": "Create order limit validation status.",
+          "enum": [
+            "Valid",
+            "Invalid",
+            "Skipped"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ValidationStatus",
+            "modelAsExtensible": true,
+            "values": [
+              {
+                "value": "Valid",
+                "description": "Validation is successful"
+              },
+              {
+                "value": "Invalid",
+                "description": "Validation is not successful"
+              },
+              {
+                "value": "Skipped",
+                "description": "Validation is skipped"
+              }
+            ]
+          }
+        }
+      },
+      "x-ms-discriminator-value": "ValidateCreateOrderLimit"
+    },
+    "DataAccountDetails": {
+      "description": "Account details of the data to be transferred",
+      "required": [
+        "dataAccountType"
+      ],
+      "type": "object",
+      "properties": {
+        "dataAccountType": {
+          "description": "Account Type of the data to be transferred.",
+          "enum": [
+            "StorageAccount",
+            "ManagedDisk"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "DataAccountType",
+            "modelAsExtensible": true,
+            "values": [
+              {
+                "value": "StorageAccount",
+                "description": "Storage Accounts ."
+              },
+              {
+                "value": "ManagedDisk",
+                "description": "Azure Managed disk storage."
+              }
+            ]
+          }
+        },
+        "sharePassword": {
```

Comment:
Mark this as x-ms-secret and put the appropriate x-ms-mutability on it to indicate it won't be returned in responses